src/main/java/gregtech/client/utils/TooltipHelper.java

```diff
@@ -43,7 +44,7 @@ public class TooltipHelper {
      * @param codes The codes, in order, that this formatting code should oscillate through. MUST be at least 2.
      */
     public static GTFormatCode createNewCode(int rate, TextFormatting... codes) {
-        if (rate <= 0) {
+        if (rate <= 0 && !ConfigHolder.client.blinkingTooltips) {
```

Comment:
I this check may break the game as you will not create valid codes but some one will still use them which mean they will try to use `null` in some way. We should allow creating such code, we just need to prevent it's effect.